hoot-core/src/main/cpp/hoot/core/schema/OsmSchema.cpp

```diff
@@ -1864,10 +1864,13 @@ QString OsmSchema::getFirstType(const Tags& tags, const bool allowGeneric)
   {
     const QString key = keys.at(i);
     const QString val = tags[key];
-    const QString kvp = toKvp(key, val);
-    if (isTypeKey(key) && (allowGeneric || !isGenericKvp(kvp)))
+    if (!val.trimmed().isEmpty())
```

Comment:
Bug fix to prevent returning tags with empty values. Somewhere in hoot during conflation empty `highway`, `bridge`, `tunnel` tags get added in...not sure where...its very annoying and has been like that for a long time.